### PR TITLE
Prepare for 0.6

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

File: gdk4/src/lib.rs [hunk 1]
```diff
@@ -2,9 +2,6 @@
 
 #![cfg_attr(feature = "dox", feature(doc_cfg))]
 #![allow(clippy::type_complexity)]
-#![allow(clippy::derive_hash_xor_eq)]
-#![allow(clippy::non_send_fields_in_send_ty)]
-#![allow(clippy::too_many_arguments)]
```
Comment (sdroege) [inline gdk4/src/lib.rs:7]: How did you find this?

Comment (bilelmoussaoui) [reply gdk4/src/lib.rs:7]: By running clippy multiple times :D

Comment (sdroege) [reply gdk4/src/lib.rs:7]: You mean you removed them and checked if it failed?

Comment (bilelmoussaoui) [reply gdk4/src/lib.rs:7]: yes

Comment (sdroege) [reply gdk4/src/lib.rs:7]: I guess that works, thanks :) I was hoping that you found an easier way

**Pull request merged**